### DISCUSSION TITLE
Lost in Translation (strange behaviour with cookies consent)

### DISCUSSION THREAD

**Discussion by davideriboli** [squidfunk/mkdocs-material#6082] [General]
Hi friends,

I'm not shure this is a real bug, so I'm asking here before opening an issue.

I'm italian, the language of my site is setted on IT and I'm using the cookie consent.

When I render the site on my pc with _mkdoc serve_, the button for "Accept" is "Accetta" and this is right.

But when I run the deploy in GitHub, the button for "Accept" is "Accettazione" and this is wrong in italian.

Any ideas?

**Comment by squidfunk** [comment]
Sounds like you're building with an outdated version of Material for MkDocs. The Italian translations were updated and corrected in 9.2.8 (via https://github.com/squidfunk/mkdocs-material/commit/bfec92b226ac794dd8f822c212697c2d58f59c53). Make sure you're using at least Material for MkDocs 9.2.8 when building on GitHub.

**Comment by davideriboli** [comment]
Thank you! I was building with an old beta...
